### ISSUE TITLE
Handskit t12 clone

### ISSUE THREAD

**Issue by keiser1080** [dreamcat4/t12-t245-controllers-docs#1]
Hi,

I received recently a t12 clone branded handskit.
The build quality seems to be good, but the soldering station kit was missing the two iron tip.
I tried first time the station using my power cord, (without the tip) the screen show error.

After that I have tried using the provided power cord and my tip.
I see a spike on the power cord, and the general fuse from my house go down.

I opened the soldering station and I see a shortcut between the aviator plug pin labeled E and pin G.

The board is very similar to the board v2 revision 1 but with a blue silk.
 I  tried the power supply using another power cord and I mesure around 23v.

Then I tried to power the controller using my bench power supply set at 24v 1A.

I have been see a blue smoke located somewhere under the screen. 
The diode labeled D2 labeled (||| T4) is  broken.
Here the list of the components:
```
U1 :
F103C8Tb
U2 :
433m2
918 no8
Q1 :
Tpc8107
D2 :
||| T4
U4 :
ATMTCO86
```

Conclusion this soldering station feel a good quality but it's lacking a quality control.

Do you know the spec of the diode d2 labeled ||| T4?
Do you think I can replace with a 1N4148 or 1N4007?

**Comment by dreamcat4** [comment]
It sounds like the PSU has an issue, which has overloaded the +24v rail on the controller PCB. So you need to test the PSU seperately and check what is going on there.

For the controller PCB... yes you should be able to replace D2 with a regular diode like either of the ones you happen to have. The thing is that if the PCU has overloaded this circuit... D2 feeds power into the 3.3v regulator. That drops the voltage down from +24v to 3.3v. For the MCU etc. So it is not clear which other component(s) may also be damaged by that.

In conclusion you are best to return the whole unit back to the store. And get your money back. Don't trust these in built PSU unless after reading about them on the forum. Sometimes they need to be checked yourself. And / or modified.

Cannot help you further here. This repository is purely for research, not troubleshooting.
